index.js

```diff
@@ -58,6 +58,30 @@ class ApplitoolsHelper extends Helper {
         await eyes.check(pageName, Target.window());
         await eyes.close();
     }
+
+    /**
+    * @param pageName {String} name of the page you want to check
+    * @param element {String} target element on the page that Applitools will focus on
+    * @param uniqueId {String} provide a unique id to combine tests into a batch
+    * @param matchLevel {String} set the match level. Possible values: Exact, Strict, Content, Layout
+    * 
+     */
+    async eyeCheckRegion(pageName, element, uniqueId, matchLevel) {
```

Comment:
aha, so we do could something like this

`
async eyeCheck({ pageName, element, uniqueId, matchLevel }) {
 ...
         await eyes.open(client, appName, pageName, windowsSize);
         element ? await eyes.check(pageName, Target.region(element)) : await eyes.check(pageName, Target.window());
         await eyes.close();
}
`

Reply:
My only question is about the `eyes.setForceFullPageScreenshot();` which I guess needs to be false for `Target.region` and true for `Target.window`. If so, the solution won't be that elegant but should still work. What do you think?

```
    async eyeCheck({ pageName, element, uniqueId, matchLevel }) {
        eyes.setApiKey(this.config.applitoolsKey);
        eyes.setForceFullPageScreenshot(TRUE/FALSE);
        
        if (uniqueId) {
            eyes.setBatch(pageName, uniqueId);
        }

        if (matchLevel) {
            eyes.setMatchLevel(matchLevel);
        }

        await eyes.open(client, appName, pageName, windowsSize);
        element ? await eyes.check(pageName, Target.region(element)) : await eyes.check(pageName, Target.window()); 
        await eyes.close(); 
    }
```

Reply:
I updated the code in the comment above

Reply:
I think it's okay to go :D

Reply:
I got stuck, to be honest. Using the code from above returns this error from the Applitools SDK: `IllegalArgument: testName is null or undefined`. I tried to play with the `eyes.setForceFullPageScreenshot()` as that's the only difference in the order of the commands but no luck.

I can propose 2 options:
1. If you have time, can you please take a look?
2. If not, maybe, we can merge it like it was before with 2 separate methods?